modules/insights-operator-advisor-overview.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_content-type: CONCEPT
+[id="insights-operator-advisor-overview_{context}"]
+= About Red Hat Insights Advisor for {product-title}
+
+You can use Insights Advisor to assess and monitor the health of your {product-title} clusters. Whether you are concerned about individual clusters, or with your whole infrastructure, it is important to be aware of your exposure to issues that can affect service availability, fault tolerance, performance, or security.
+
+Insights repeatedly analyzes the data Insights Operator sends using a database of *Recommendations*, which are sets of conditions that can leave your {product-title} clusters at risk. Your data is then uploaded to the Insights Advisor service on Red Hat Hybrid Cloud Console where you can perform the following actions:
```

Comment:
Using "that" in this case can improve readability; when at first I read it, I was trying to figure out if "data" was part of the Operator name with missed casing (my mind can wander).
```suggestion
Insights repeatedly analyzes the data that Insights Operator sends using a database of *Recommendations*, which are sets of conditions that can leave your {product-title} clusters at risk. Your data is then uploaded to the Insights Advisor service on Red Hat Hybrid Cloud Console where you can perform the following actions:
```

Also wondering about the bolding of Recommendations. I'm unfamiliar, so if it is part of the console UI then it makes sense, but if it is introducing a concept, you might consider underscores instead (as suggested in our OpenShift docs [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference)). Also, if not a UI element, consider using lowercase "recommendations" to match casing in the next module.

Reply:
It is a UI element, but the way I word it here is more conceptual. I will just unbold it.